test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -128,10 +128,15 @@ private Map<String, Object> getPravegaOnlyDeployment(String zkLocation, int cont
         final Map<String, Object> pravegaSpec = ImmutableMap.<String, Object>builder().put("controllerReplicas", controllerCount)
                 .put("segmentStoreReplicas", segmentStoreCount)
                 .put("debugLogging", true)
-                .put("cacheVolumeClaimTemplate", pravegaPersistentVolumeSpec)
-                .put("controllerResources", getResources("2000m", "2Gi", "1000m", "2Gi"))
-                .put("segmentStoreResources", getResources("2000m", "6Gi", "1000m", "6Gi"))
-                .put("options", props)
+                .put("controllerResources", getResources(resourceWrapper.getControllerProperties().getControllerResources().getLimits().get("cpu"),
+                        resourceWrapper.getControllerProperties().getControllerResources().getLimits().get("memory"),
+                        resourceWrapper.getControllerProperties().getControllerResources().getRequests().get("cpu"),
```

Comment:
Dont pass cpu and memory  just call get resources with empty parameter and do all stuff inside the getResources() method because resource Wrapper is present there as well.

Reply:
I don't think so that's feasible. the getResources() method in the AbstractService class has 4 parameters passed and couldn't be passed empty.

Reply:
we can refactor it

Reply:
I don't think refactoring would make any sense here. The only thing we can do then is to remove that method from AbstractService class and pass the value as we did for others.